documentation/views/asset-data.rst

```diff
@@ -42,6 +42,39 @@ Finally, it is possible to set custom titles for any sensor graph by clicking on
 Internally, the asset has a `sensors_to_show`` field, which controls which sensor data appears in the plot. This can also be set by a script. Accepted formats are simple lists of sensor IDs (e.g. `[2, [5,6]]` or a more expressive format (e.g. `[{"title": "Power", "sensor": 2}, {"title": "Costs", "sensors": [5,6] }`). 
 
 
+Editing Asset FlexContext 
+=========================
+
+The Asset's ``FlexContext`` allows you to define certain fields and their values. Initially, no fields are set.
```

Comment:
"certain fields and their values" is too vague. I'd do this:

"Per asset, you can set fields in :ref:`the flex-context <flex_context>`, which will influence how scheduling works on this asset.  The flex context dialogue allows you to define either fixed values or sensors (for dynamic values / time series). Initially, no fields are set."